packages/components/src/combobox-control/index.js

```diff
@@ -0,0 +1,151 @@
+/**
+ * External dependencies
+ */
+import { useCombobox } from 'downshift';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import { Button, Dashicon } from '../';
+
+const itemToString = ( item ) => item && item.name;
+// This is needed so that in Windows, where
+// the menu does not necessarily open on
+// key up/down, you can still switch between
+// options with the menu closed.
```

Comment:
I don't think we want this functionality for this component. If we're following Example 1 from [these patterns](https://www.w3.org/TR/wai-aria-practices/examples/combobox/aria1.1pattern/listbox-combo.html) - which I assume we are as we're matching it in everything else - then up and down arrows should do nothing if the dropdown is closed.